packages/client/src/runtime/core/types/exported/ObjectEnums.ts

```diff
@@ -44,17 +44,20 @@ class NullTypesEnumValue extends ObjectEnumValue {
 }
 
 class DbNull extends NullTypesEnumValue {
-  private readonly _brand_DbNull!: void
+  // eslint-disable-next-line no-unused-private-class-members
```

Comment:
Could we add a little more comment next to this disable lint so we'll remember why we did this in. few months from now without finding this PR? :)